src/utils/system-validation.ts

```diff
@@ -0,0 +1,14 @@
+import chalk from "chalk";
+import { execa } from "execa";
+
+export const validateFoundryUp = async () => {
+  try {
+    await execa("foundryup", ["-h"]);
+  } catch (error) {
+    const message = `${chalk.bold.yellow("NOTE: Foundryup is not installed in your system.")}
+${chalk.bold.yellow("To use foundry, please install foundryup")}
```

Comment:
Lol not sure if yellow is better option for this all, but red seems too harsh but we could use red to since its an error

Reply:
I think yellow + cli exiting is good enough :P